ci/prow/templates/general_prow_config.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright [[.Year]] The Knative Authors
```

Comment:
I think "prow_config_header.yaml" would be better.

Reply:
done